Makefile

```diff
@@ -158,6 +158,9 @@ LIBSECCOMP := sysbox-libs/libseccomp/src/.libs/libseccomp.a
 LIBSECCOMP_DIR := sysbox-libs/libseccomp
 LIBSECCOMP_SRC := $(shell find $(LIBSECCOMP_DIR)/src 2>&1 | grep -E '.*\.(c|h)')
 LIBSECCOMP_SRC += $(shell find $(LIBSECCOMP_DIR)/include 2>&1 | grep -E '.*\.h')
+LIBSECCOMP_UID := $(shell stat -c %u ./sysbox-libs/libseccomp/README.md)
```

Comment:
Wondering why we need a `LIBSECCOMP_UID` and `LIBSECCOMP_GID` here? Why aren't `HOST_UID` and 'HOST_GID` enough in this case?

Reply:
I did it this way to keep the ownership of libseccomp as it was before the build. That may or may not be the same as the uid:gid of the user that is doing the build itself (normally it is, but it's not necessarily the case).